src/NeoSharp.Core/Messaging/MessageHandlerProxy.cs

```diff
@@ -48,10 +48,10 @@ public async Task Handle(Message message, IPeer sender)
             await specificMessageHandler.Handle(specificMessage, sender);
             var completedAt = DateTime.UtcNow;
 
-            var handledWithin = (completedAt - startedAt).TotalMilliseconds;
+            var handledWithin = (completedAt - startedAt).TotalSeconds;
```

Comment:
why this change?

Reply:
Because is readable in seconds for humans
with second we will get the 99.9999 and it's easier to read in the logs.

Reply:
Usually this should be very fast so we need the time in ms ... if we have some message who spend more than 1 second, we have something very wrong

Reply:
well .. that is the reason to change .... to popup the problem. It's easy to see by human eye